test/drawers/drawerTests.ts

```diff
@@ -22,17 +21,17 @@ class MockAnimator implements Plottable.Animator {
 }
 
 function createMockDrawer(dataset: Plottable.Dataset) {
-  var drawer = new Plottable.Drawer(dataset);
+  const drawer = new Plottable.Drawer(dataset);
```

Comment:
Let's take the following dummy suggestion

```typescript
const x = [];
x.push(2);
```

`x` changed fundamentally, yet it is not clear to the reader who assumes const gives safety. Hence, `const` does not give safety.

Recapping: using const:
- Gives no safety, even worse it misleads the reader into thinking that the state of the variable will not change
- Introduces mental overhead. Every time you have to think what will be the usage of the variable

Suggestion:
- Use `const` only for primitive types. Or in the cases where you would normally TYPE_THE_OBJECT_ALL_CAPS_CUZ_ITS_CONST

Aside:
- We are diverging. We were supposed to do this change only because `let` > `var` because of block scope. Now we are doing a completely different change.